lib/evss/auth_headers.rb

```diff
@@ -37,11 +37,20 @@ def eauth_json
           edi: @user.edipi,
           firstName: @user.first_name,
           lastName: @user.last_name,
-          birthDate: iso8601_birth_date
+          birthDate: iso8601_birth_date,
+          gender: gender
         }
       }.to_json
     end
 
+    def gender
+      if @user.gender == 'F'
+        'FEMALE'
+      elsif @user.gender == 'M'
+        'MALE'
+      end
```

Comment:
Maybe a guard clause for the case where `gender` is `nil` or `""`?
```ruby
return '' if @user.gender.blank?
```

Reply:
I figured in that case it would automatically return `nil` and pass that through. Is there a reason why empty string is preferable?

Reply:
You could also use a case... might be cleaner:
```
case @user.gender
  when 'F'
    'FEMALE'
  when 'M'
    'MALE'
  else
    ''
end
```
But thats not answer to `nil` vs `""` :)

Reply:
👍 to case. On `nil` (or `null` once to_json'ed) vs `''`. I know that a `nil` value for the header `va_eauth_birlsfilenumber` caused an issue before, probably best to check with EVSS on what they're expecting if gender is unavailable.

Reply:
I don't think it much matters what's passed in when gender is unavailable because anything other than `FEMALE` or `MALE` will error out. I've been told that if gender is unavailable EVSS expects the user to go to PCIU and enter a gender there, and then come back to the 526 form. 😐